blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitPipelineCreateRequest.java

```diff
@@ -40,6 +42,10 @@ public GitPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
 
     @Override
     public BluePipeline create(Reachable parent) throws IOException {
+        User authenticatedUser =  User.current();
+        if(authenticatedUser == null){
+            throw new ServiceException.UnauthorizedException("Must login to create a pipeline");
+        }
```

Comment:
You probably will need some additional checks here, such as:

* Did the system admin disable the per-user credentials store?
<img width="878" alt="turning off user credentials store" src="https://cloud.githubusercontent.com/assets/209336/22500739/b8081592-e85d-11e6-9793-d8e34128f34b.png">

* Did the system admin disable this Blue Ocean User Credentials Proxy/Gateway credential store

* Did the system admin block storage of the credential type we need to store in the user credentials store
<img width="885" alt="screen shot 2017-02-01 at 09 08 01" src="https://cloud.githubusercontent.com/assets/209336/22500802/f7d23f36-e85d-11e6-9f66-e3fe31d667d1.png">

Probably not a blocker for merging this, but rather something you will need to be aware of and decide what you want to do about it and how to behave if the system admin has made configuration choices about credentials storage that prevent the blue ocean pipeline functionality from working